app/models.py

```diff
@@ -30,6 +32,8 @@ class Service(db.Model):
                                nullable=False)
     data = db.Column(JSON)
 
+    status = db.Column(db.String, index=False, unique=False, nullable=False)
```

Comment:
Shouldn't status be indexed? It seems that we're going to filter enabled/published services on almost every request.